src/test/scala/AWSKeySigningSpec.scala

```diff
@@ -22,23 +28,27 @@ class AWSKeySigningSpec extends FunSpec with Matchers {
       val payload = "Action=ListUsers&Version=2010-05-08"
       val hashedPayload = SignRequestForAWS.signPayload(payload)
       hashedPayload shouldBe "b6359072c78d70ebee1e81adcbab4f01bf2c23245fa365ef83fe8f1f955085e2"
+      println(SignRequestForAWS.signPayload("UserName=NewUser&Action=CreateUser&Version=2010-05-08"))
```

Comment:
Should I log the json responses? They are not explicitly tested just the status code.